tensorflow/python/keras/callbacks.py

```diff
@@ -1378,6 +1378,29 @@ class TensorBoard(Callback):
 
   You can find more information about TensorBoard
   [here](https://www.tensorflow.org/get_started/summaries_and_tensorboard).
+  
+  Example:
+
+  ```python
+  # Import the datetime module to display the date and time
+  import datetime
```

Comment:
Can you remove the datetime logic? Let's just show the minimum example for this callback